definitions/npm/luxon_v1.x.x/flow_v0.32.0-/luxon_v1.x.x.js

```diff
@@ -504,6 +504,7 @@ declare module "luxon" {
       options?: IntlDateTimeFormatOptions
     ): Array<{ type: string, value: number }>;
     toLocaleString(options?: IntlDateTimeFormatOptions): string;
+    toMillis(): number;
```

Comment:
This method exists from version `1.2.1` so it might need a different definition file? https://github.com/moment/luxon/blob/master/changelog.md

Reply:
i don't know how to add version `>=1.2.1` 😢

Reply:
Create a new file under the path `definitions/npm/luxon_v1.2.x/flow_v0.32.0-/luxon_v1.2.x.js`. You'll also need to choose to change `1.x.x` to `1.1.x`.

Reply:
yes but it's contains 1.2.0 also

Reply:
Yeah, this is one of the major downsides of handling multiple versions of a definition.

Reply:
fixed